homeassistant/components/fressnapf_tracker/switch.py

```diff
@@ -14,6 +14,9 @@
 from . import FressnapfTrackerConfigEntry
 from .entity import FressnapfTrackerEntity
 
+# Coordinator is used to centralize the data updates
```

Comment:
Same here... what about switch actions?

Reply:
Should have read the documentation better. Thanks!